.expeditor/config.yml

```diff
@@ -1,6 +1,30 @@
+# Documentation available at https://expeditor.chef.io/docs/getting-started/
 ---
+# Slack channel in Chef Software slack to send notifications about build failures, etc
+slack:
+  notify_channel: chef-found-notify
+
 github:
   delete_branch_on_merge: true
 
 pipelines:
-  - verify
+  - verify:
+      public: true
+  - habitat/build
+
+promote:
+  channels:
+    - unstable
```

Comment:
Do we want/need a `current` channel here? Not sure if we are trying to match our core products like `chef-infra` which will depend on this package.

Reply:
Typically we don't add current channels for non-omnibus projects. We can always add it later if we want to.